app/views/profiles/show.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="row">
+  <div class="medium-4 columns">
+    <div>
+      <%= render "profile" %>
+    </div>
+  </div>
+  <div class="medium-4 columns">
+    <div>
+      <%= render "change_password" %>
+    </div>
+  </div>
+  <div class="medium-4 columns">
+    <div>
+      <div class="medium-12 columns">
+        <h3>Reset Your Private Key</h3>
```

Comment:
I did it this way because that is how it is set up in account_manage. I agree that it would be better to put that stuff in generate_key, so I'll do that.